README.md

```diff
@@ -99,20 +134,40 @@ Manage traditional [variants] of a configuration - like
 _development_, _staging_ and _production_ - using
 [overlays] that modify a common [base].
 
-![overlay image][imageOverlay]
+```
+
+overlay: kustomization + patches + more resources
+
+kustomization.yaml                              replica_count.yaml                     cpu_count.yaml
++---------------------------------------+       +-------------------------------+      +------------------------------------------+
+| commonLabels:                         |       | apiVersion: apps/v1           |      | apiVersion: apps/v1                      |
+|   app: myapp                          |       | kind: Deployment              |      | kind: Deployment                         |
+| resources:                            |       | metadata:                     |      | metadata:                                |
+|   - ../../base                        |       |   name: myapp                 |      |   name: myapp                            |
+| patches:                              |       | spec:                         |      | spec:                                    |
+|   - path: replica_count.yaml          |       |   spec:                       |      |   spec:                                  |
```

Comment:
- We still need to reference the cpu count patch in the patches list
- `spec.spec.replicas` should just be `spec.replicas` in the middle document
- `spec.spec.containers` should be `spec.template.spec.containers` in the rightmost document